client/verta/verta/_uac/_workspace.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from verta._protos.public.uac import WorkspaceV2_pb2 as _Workspace
```

Comment:
See my comment in `_organization.py`.

```suggestion
from verta._protos.public.uac import WorkspaceV2_pb2
```